verilog/tools/kythe/scope_resolver.h

```diff
@@ -180,10 +181,9 @@ class ScopeResolver {
   ScopeResolver(const ScopeResolver&) = delete;
   ScopeResolver(ScopeResolver&&) = delete;
   ScopeResolver& operator=(const ScopeResolver&) = delete;
-  ScopeResolver& operator=(ScopeResolver&) = delete;
```

Comment:
From context, this was probably intended to be move-assign:

```c++
ScopeResolver& operator=(ScopeResolver&&) = delete;
```

Reply:
Ah, makes sense. Done.